addon/-private/system/model/model.js

```diff
@@ -456,7 +459,7 @@ const Model = EmberObject.extend(Evented, {
   */
   toJSON(options) {
     // container is for lazy transform lookups
-    let serializer = this.store.serializerFor('-default');
+    let serializer = this._internalModel.store.serializerFor('-default');
```

Comment:
Why do we not lookup store directly anymore?

Reply:
We only accessed `store` two places from `Model` ourselves. Moving these two places to `_internalModel` limits our exposure. We now no longer care whether `store` is properly assigned or not. We give it to the model but end users may override as it is not necessary to stay there.